server/boot/root.js

```diff
@@ -5,8 +5,11 @@ module.exports = function(server) {
   var router = server.loopback.Router();
   router.get('/', (req, res) => {
     const user = {
-      firstName: "test",
-      lastName: "testerson"
+      token: null,
+      userId: null,
+      admin: false
+      // firstName: "test",
```

Comment:
Remove lines 11 & 12 if you're not using them

Reply:
Removed.